src/query-builder/utils/queryStringProcessor.ts

```diff
@@ -82,11 +82,11 @@ const getEmptyClause = (id: number): Clause => ({
   logicOperator: Operator.AND,
 });
 
-export const parse = (queryString = ''): Clause[] => {
+export const parse = (queryString = '', startId = 0): Clause[] => {
```

Comment:
Could you add a comment about what the `startId` represents within the context of parsing the query string?

Reply:
Done

Reply:
Closing now so that it goes in the release